generators/client/index.js

```diff
@@ -187,19 +200,30 @@ module.exports = JhipsterClientGenerator.extend({
             insight.track('generator', 'app');
             insight.track('app/useSass', this.useSass);
             insight.track('app/enableTranslation', this.enableTranslation);
+            insight.track('app/nativeLanguage', this.nativeLanguage);
+            insight.track('app/languages', this.languages);
         },
 
         configureGlobal: function () {
             // Application name modified, using each technology's conventions
             this.angularAppName = this.getAngularAppName();
-            this.camelizedBaseName = _.camelize(this.baseName);
-            this.slugifiedBaseName = _.slugify(this.baseName);
+            this.camelizedBaseName = _s.camelize(this.baseName);
+            this.slugifiedBaseName = _s.slugify(this.baseName);
             this.lowercaseBaseName = this.baseName.toLowerCase();
+            this.nativeLanguageShortName = this.enableTranslation ? this.nativeLanguage.split("-")[0] : 'en';
         },
 
         saveConfig: function () {
             this.config.set('useSass', this.useSass);
-            this.config.set('enableTranslation', this.enableTranslation);
+            if (this.enableTranslation === true) {
```

Comment:
Any specific reason to check for true and false like this?  Shouldn't just `if (this.enableTranslation) {} else {}` do?